addon/services/ajax.ts

```diff
@@ -0,0 +1,11 @@
+import Service from '@ember/service';
+import AjaxRequestMixin from '../mixins/ajax-request';
+
+export default Service.extend(AjaxRequestMixin);
+
+// DO NOT DELETE: this is how TypeScript knows how to look up your services.
+declare module '@ember/service' {
+  interface Registry {
+    ajax: any;
```

Comment:
I couldn't get this right, so I just set it to `any` for now.

If I export the service like this instead:

```typescript
export default class AjaxService extends Service.extend(AjaxRequestMixin);
```

some of my tests started complaining that I wasn't using `new` with the class.